<?xml version="1.0"?>
<launch>
	<node name="nodo_teleop" pkg="teleop_twist_keyboard" type="teleop_twist_keyboard.py" output="screen">
		<remap from="/cmd_vel" to="/aux_cmd_vel"/>
	</node>
</launch>
